debian/control

```diff
@@ -8,6 +8,7 @@ Build-Depends:
   cargo,
   libudev-dev,
   libinput-dev,
+  playerctl,
```

Comment:
This shouldn't be in `Build-Depends`.

Either `Depends` or `Recommends` works. Though `Recommends` may be better so it can be removed if someone really wants. It probably doesn't matter much.

Reply:
Should be good now!